src/devices/device_models/common/rating_constraints.jl

```diff
@@ -36,7 +36,7 @@ function norm_two_constraint(ps_m::CanonicalModel,
 
     for r in rating_data
         for t in time_steps
-          constraint[r[1], t] = JuMP.@constraint(ps_m.JuMPmodel, var1[r[1], t] + var2[r[1], t] <= r[2].max^2)
+          constraint[r[1], t] = JuMP.@constraint(ps_m.JuMPmodel, var1[r[1], t]^2 + var2[r[1], t]^2 <= r[2].max^2)
```

Comment:
There is something odd in this implementation probably from before. 
1. It shouldn't be called the norm_two_constraint since it doesn't use the two norm formulation. It is a quadratic form. 
2. The rating data should be only `Vector{Tuple{String, Float}}` I think we call it `NamedTuple` and the constraint should be 
```
 JuMP.@constraint(ps_m.JuMPmodel, var1[r[1], t]^2 + var2[r[1], t]^2 <= r[2]^2
```